.github/workflows/macos.yml

```diff
@@ -0,0 +1,34 @@
+# Workflow for macOS (latest).
+name: macOS
+
+on:
+  push:
+    branches:
+      - "**"
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: macos-latest
+    steps:
+      # Checkout the repository under $GITHUB_WORKSPACE.
+      - uses: actions/checkout@v1
```

Comment:
Why `v1` and not `v2`?

And let's leave a comment why we're not recursively checking out submodules. 

Here and in all the other files too.